### PR TITLE
[4.4] Cypress small update README.md (2)

### PR DESCRIPTION
**Pull request by sandewt**
Pull Request for Issue # .

small update README.md

Related to #40175

### CODE REVIEW

File: tests/System/README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ To Create new tests, create a cy.js file in a new folder which matches the follo
 Probably the easiest way is to copy an existing file and adapt it to the extension which should be tested.
 
 ## Some developer information
-Tests should be
+Tests should be:
```
Comment (laoneo) [inline tests/System/README.md:34]: If you add here a double point, then I guess you need to upper case the first character in the list entries.

File: tests/System/README.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ A couple of steps are needed before the CMS system tests can be executed on the
    1. `composer install`
    2. `npm ci`
 3. Copy the cypress.config.dist.js to cypress.config.js in the root of the joomla folder
-4. Adapt the env variables in the file cypress.config.js, they need to point to a database server where joomla can be installed
+4. Adjust the baseUrl and adapt the env variables in the file cypress.config.js, they need to point to a database server where joomla can be installed
```
Comment (laoneo) [inline tests/System/README.md]: I would add here a new list item with the baseUrl and what it is for.

File: tests/System/README.md (hunk 1 above)
Comment (sandewt) [reply tests/System/README.md:34]: ~Below in the text no upper case (capital) letters are used in the list either.~

Capital letters are also used above. To be consistent, I am also adjusting this here.
Whether a capital letter is used may even depend on the language. @brian

[EDIT]

File: tests/System/README.md (hunk 2 above)
Comment (sandewt) [reply tests/System/README.md]: Done, I hope the content is correct and it is a little understandable English.

**Comment by sandewt** [comment]
Thanks all

**Pull request merged**